client/ayon_maya/plugins/create/create_review.py

```diff
@@ -35,69 +31,18 @@ class CreateReview(plugin.MayaCreator):
     useMayaTimeline = True
     panZoom = False
 
-    # Overriding "create" method to prefill values from settings.
-    def create(self, product_name, instance_data, pre_create_data):
-
-        members = list()
-        if pre_create_data.get("use_selection"):
-            members = cmds.ls(selection=True)
-
-        project_name = self.project_name
-        folder_path = instance_data["folderPath"]
-        task_name = instance_data["task"]
-        folder_entity = ayon_api.get_folder_by_path(
-            project_name, folder_path, fields={"id"}
-        )
-        task_entity = ayon_api.get_task_by_name(
-            project_name, folder_entity["id"], task_name, fields={"taskType"}
-        )
-        preset = lib.get_capture_preset(
-            task_name,
-            task_entity["taskType"],
-            product_name,
-            self.project_settings,
-            self.log
-        )
-        self.log.debug(
-            "Using preset: {}".format(
-                json.dumps(preset, indent=4, sort_keys=True)
-            )
-        )
-
-        with lib.undo_chunk():
-            instance_node = cmds.sets(members, name=product_name)
-            instance_data["instance_node"] = instance_node
-            instance = CreatedInstance(
-                self.product_type,
-                product_name,
-                instance_data,
-                self)
-
-            creator_attribute_defs_by_key = {
-                x.key: x for x in instance.creator_attribute_defs
-            }
-            mapping = {
-                "review_width": preset["Resolution"]["width"],
-                "review_height": preset["Resolution"]["height"],
-                "isolate": preset["Generic"]["isolate_view"],
-                "imagePlane": preset["ViewportOptions"]["imagePlane"],
-                "panZoom": preset["Generic"]["pan_zoom"]
-            }
-            for key, value in mapping.items():
-                creator_attribute_defs_by_key[key].default = value
-
-            self._add_instance_to_context(instance)
+    @classmethod
```

Comment:
Why is this classmethod?

Reply:
My bad! I was clearly sleeping when drafting this PR.